src/runtime/components/nuxt-img.vue

```diff
@@ -1,9 +1,5 @@
 <template>
-  <img
-    :key="nSrc"
-    :src="nSrc"
-    v-bind="nAttrs"
-  >
+  <img :key="nSrc" :src="nSrc" v-bind="nAttrs" v-on="listeners" />
```

Comment:
Is it maybe possible directly using `$listeners` here without creating a computed + new object? Also would be nice adding same fix to `nuxt-picture` :)

Reply:
@pi0 I plan on doing these changes sometime soon (also the docs and tests @shadow81627 mentioned) - keep you posted